source/__tests__/integration/import-profile.test.js

```diff
@@ -13,9 +12,19 @@ describe('imports and edits a v0.0.2 profile from a json file', () => {
     return await expect(page).toClick(profileLoadedSel)
   })
 
-  it('has expected profile attribute values and resource templates', async () => {
-    expect.assertions(10) // includes 2 in beforeAll
+  test('profile attribute values', async () => {
```

Comment:
what follows here is a slight reorg of some tests - trying to make them shorter and clearer.